docs/docusaurus/docs/core/connect_to_data/sql_data/_create_a_data_asset/create_a_data_asset.py

```diff
@@ -6,16 +6,15 @@
 pytest  --postgresql --docs-tests -k "create_a_data_asset_postgres" tests/integration/test_script_runner.py
 """
 
-# This section is setup for the environment used in the example script.
-from tests.integration.db.taxi_data_utils import load_data_into_test_database
-
-# add test_data to database for testing
-load_data_into_test_database(
-    table_name="postgres_taxi_data",
-    csv_path="./data/yellow_tripdata_sample_2019-01.csv",
-    connection_string="postgresql+psycopg2://postgres:@localhost/test_ci",
+# The following import and setup method sets up the data and environment used to test this example.
+# It can be disregarded by anyone using referencing this script as an example of a
+# GX workflow.
+from docs.docusaurus.docs.components._testing.utility_scripts.postgres_data_setup import (
+    setup,
 )
 
+setup()
```

Comment:
This validates that the snippet runs without dying. You may want to put in an assert after each snippet ends to so some validation that we are seeing what we expect.

For example after the `retrieve data source` snippet you could add `assert data_source.name == datasource_name`